### DISCUSSION TITLE
Games vs. Sandboxie Plus (moved to discussions/4172)

### DISCUSSION THREAD

**Discussion by ghost** [sandboxie-plus/Sandboxie#2927] [Documents and notes]
# Compatibility with games and applications
The list features games and applications which have been tested with Sandboxie Plus. The table provides detailed info about each application, environment it was tested in and reported issues. In order to find out any discrepancies, listed applications have been tested in [Standard (![SbieCtrl_TRAYICON_EMPTY](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/d08ad3c1-c4e3-4a91-a373-475bb0d6726b)yellow) box](https://sandboxie-plus.com/box-preset-comparison/) and in [Application Compartment (![sandbox-g-empty](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/07c0361c-ed29-4090-8bcd-5d4d877f1e68)green) box](https://sandboxie-plus.com/compartment-mode/). Lastly, some applications have been tested in order to find out performance difference between the aforementioned box types.
<hr>

:pushpin: Table legend:
:repeat: - repetition, the same issue as in the previous box
<hr>:arrow_double_down:APPS

| Title | ![SbieCtrl_TRAYICON_EMPTY](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/d08ad3c1-c4e3-4a91-a373-475bb0d6726b) Standard | ![sandbox-g-empty](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/07c0361c-ed29-4090-8bcd-5d4d877f1e68) Application Compartment | ![SandMan_IDI_ICON1](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/081e9dec-9e7f-4fdc-a1cc-cf6c2511adcd) Version | :computer: OS
| :---: | :---: | :---: | :---: | :---: |
| [EA App](https://www.ea.com/ea-app/) | :x:<br>#3260 | :x:<br>:repeat: | 1.11.0 | Win7x64
| [Epic Games Launcher](https://store.epicgames.com/download/) | :white_check_mark:<br>#3261 | :white_check_mark:<br>:repeat: | 1.13.7 | Win7x64
| [GOG Galaxy](https://www.gog.com/galaxy) | :x:<br>#3297<br>#1246 | :white_check_mark: | 1.13.7 | Win7x64
| [Steam](https://store.steampowered.com/about/) | :white_check_mark:<br>#3484 | :white_check_mark: | 1.13.7 | Win7x64
| [Ubisoft Connect](https://ubisoftconnect.com/) | :white_check_mark: | :white_check_mark: | 1.11.0 | Win7x64
<br>
:arrow_double_down:GAMES

| Title | ![SbieCtrl_TRAYICON_EMPTY](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/d08ad3c1-c4e3-4a91-a373-475bb0d6726b) Standard | ![sandbox-g-empty](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/07c0361c-ed29-4090-8bcd-5d4d877f1e68) Application Compartment | ![SandMan_IDI_ICON1](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/081e9dec-9e7f-4fdc-a1cc-cf6c2511adcd) Version | :computer: OS
| :---: | :---: | :---: | :---: | :---: |
| [Amerzone: The Explorer's Legacy](https://www.mobygames.com/game/3973/amerzone-the-explorers-legacy/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [Arma: Cold War Assault](https://www.mobygames.com/game/53252/arma-cold-war-assault/)<br>• GOG | :white_check_mark: | :white_check_mark: | 1.10.2 | Win7x64
| [Battlefield 2](https://www.mobygames.com/game/18194/battlefield-2/)<br>• Retail<br>• BF2Hub | :white_check_mark:<br>#3136<br>:warning: _Block Internet Access_ preset causes the game to crash | :white_check_mark:<br>:repeat:<br>:repeat: | 1.11.0 | Win7x64
| [Catan: Die Erste Insel](https://www.mobygames.com/game/16206/catan-die-erste-insel/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [Cue Club 2](https://www.mobygames.com/game/102939/cue-club-2/)<br>• Steam | :white_check_mark: | :white_check_mark: | 1.10.2 | Win7x64
| [Devastation](https://www.mobygames.com/game/8823/devastation/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.9.7 | Win7x64
| [Fast Food Tycoon](https://www.mobygames.com/game/2864/fast-food-tycoon/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [GRID 2](https://www.mobygames.com/game/60730/grid-2/)<br>• Steam | :x:<br>#3218 | :x:<br>:repeat:  | 1.11.0 | Win7x64
| [GRID: Autosport](https://www.mobygames.com/game/72953/grid-autosport/)<br>• Steam | :white_check_mark: | :white_check_mark: | 1.12.9 | Win7x64
| [Nosferatu: The Wrath of Malachi](https://www.mobygames.com/game/12130/nosferatu-the-wrath-of-malachi/)<br>• Retail<br>• GOG | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [Operation Flashpoint GOTY](https://www.mobygames.com/game/12132/operation-flashpoint-game-of-the-year-edition/)<br>• GOG | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [Rainbow Six: Rogue Spear](https://www.mobygames.com/game/964/tom-clancys-rainbow-six-rogue-spear/)<br>• Black Ops 2.0 | :white_check_mark: | :white_check_mark: | 1.14.3 | Win7x64
| [RC Cars](https://www.mobygames.com/game/12554/rc-cars/)<br>• GOG | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [Söldner: Secret Wars](https://www.mobygames.com/game/15230/soldner-secret-wars/)<br>• Community Edition | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [The Sacred Amulet](https://www.mobygames.com/game/3770/the-sacred-amulet/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.11.2 | Win7x64
| [The Sting!](https://www.mobygames.com/game/4295/the-sting/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [TimeScape: Journey to Pompeii](https://www.mobygames.com/game/2482/timescape-journey-to-pompeii/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [Traitors Gate](https://www.mobygames.com/game/1863/traitors-gate/)<br>• Retail | :x:<br>:warning: QuickTime (version 2, 3 and 4) doesn't work. It is required to install QuickTime 7.7.9 or QuickTime Alternative 3.2.2<br>:warning: QuickTime videos are laggy<br>• Launches instantly | :white_check_mark:<br>• QuickTime (version 2, 3 and 4) works fine but it is recommended to install QuickTime 7.7.9 or QuickTime Alternative 3.2.2<br>• QuickTime videos work fine<br>:warning: Takes 60s to launch | 1.12.7 | Win7x64
| [Traitors Gate 2](https://www.mobygames.com/game/10965/traitors-gate-2-cypher/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.14.6 | Win7x64
| [Unreal Gold](https://www.mobygames.com/game/4892/unreal-gold/)<br>• Retail<br>• GOG<br>• OldUnreal227j | :white_check_mark: | :white_check_mark: | 1.9.3 | Win7x64
| [Unreal Tournament GOTY](https://www.mobygames.com/game/7158/unreal-tournament-game-of-the-year-edition/)<br>• Retail<br>• OldUnreal-UTPatch469d | :white_check_mark: | :white_check_mark: | 1.11.0 | Win7x64
| [World Racing 2](https://www.mobygames.com/game/22600/world-racing-2/)<br>• Steam | :white_check_mark: | :white_check_mark: | 1.10.2 | Win7x64
| [WWII Tank Commander](https://www.mobygames.com/game/18686/wwii-tank-commander/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.10.3 | Win7x64
<br>
:arrow_double_down:BENCHMARK

| Title | ![SbieCtrl_TRAYICON_EMPTY](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/d08ad3c1-c4e3-4a91-a373-475bb0d6726b) Standard | ![sandbox-g-empty](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/07c0361c-ed29-4090-8bcd-5d4d877f1e68) Application Compartment | :open_file_folder: Unsandboxed | ![SandMan_IDI_ICON1](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/081e9dec-9e7f-4fdc-a1cc-cf6c2511adcd) Version | :computer: OS
| :---: | :---: | :---: | :---: | :---: | :---: |
| [GRID 2](https://www.mobygames.com/game/60730/grid-2/)<br>• Steam | min_fps="105.671944"<br>max_fps="159.882614"<br>av_fps="129.760391" | min_fps="137.505951"<br>max_fps="237.419540"<br>av_fps="173.295853  | N/A | 1.12.6 | Win7x64
| [GRID: Autosport](https://www.mobygames.com/game/72953/grid-autosport/)<br>• Steam | min_fps="83.408981"<br>max_fps="113.926949"<br>av_fps="101.507591" | min_fps="115.546089"<br>max_fps="203.296005"<br>av_fps="150.188889"  | N/A | 1.12.6 | Win7x64
| [Operation Flashpoint GOTY](https://www.mobygames.com/game/12132/operation-flashpoint-game-of-the-year-edition/)<br>• GOG | 2449 points | 2548 points | 5034 points | 1.12.7 | Win7x64

**Comment by e-t-l** [comment]
Can Steam itself be installed inside Sandboxie?

**Reply by Domochevsky** [reply]
Yes, but it does require admin privileges at all times, otherwise it goes into some kind of interface refresh loop.

**Reply by e-t-l** [reply]
Interesting. Do you know if it needs true admin access or if dropadminrights=y will suffice?

**Reply by Domochevsky** [reply]
Dropping admin rights while using an admin account will cause the refresh loop to happen, from personal experience. However, running it with dropped admin rights, but an Application Compartment sandbox (instead of standard isolation) causes it to run just fine.

**Reply by ghost** [reply]
Steam is not supposed to be launched as Admin (regardless of being launched in sandbox or in real OS) due to security reasons.

**Reply by dzadonskii** [reply]
Steam works fine in Data Protection (Blue) sandbox if you start it without CEF sandbox feature by calling steam.exe with -cef-disable-sandbox parameter. I suppose it should also work in Standard (Yellow) this way.

Updating comment to include my config for Blue sandbox:

```
Enabled=y
BlockNetworkFiles=y
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Desktop%
BorderColor=#f88501,ttl,6
Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=AutoRecoverIgnore
ConfigLevel=10
UseFileDeleteV2=y
UseRegDeleteV2=y
AutoRecover=n
UsePrivacyMode=y
AllowNetworkAccess=!<InternetAccess>,n
DropAdminRights=y
FakeAdminRights=y
ClosePrintSpooler=y
ProtectHostImages=y
ProcessGroup=<InternetAccess>,steamwebhelper.exe,steam.exe,mgsvtpp.exe
PromptForInternetAccess=y
```

**Reply by e-t-l** [reply]
I was able to install Steam in a standard yellow Sandbox and run it with little issue, and I do not need to run it as admin or disable CEF. So it's definitely possible. (I'm on my phone right now but I'll update my comment to include my sb config later) 

config update:
```
Enabled=y
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
BorderColor=#02f6f6,ttl,6
Template=AutoRecoverIgnore
Template=LingerPrograms
Template=qWave
Template=FileCopy
Template=SkipHook
Template=OpenBluetooth
Template=RemoteDesktop
ConfigLevel=10
UseFileDeleteV2=y
UseRegDeleteV2=y
AutoRecover=y
ForceProcess=Steam.exe
BlockNetParam=n
OpenDevCMApi=y
MsiInstallerExemptions=y
DisableRTBlacklist=y
NetworkAccess=*,Allow;Port=9296;Protocol=UDP
NetworkAccess=*,Allow;Port=9297;Protocol=UDP
NetworkAccess=*,Allow;Port=9303;Protocol=UDP
NetworkAccess=remoteplay-version-patcher.exe,Allow;Port=*;Address=*;Protocol=Any
SpecialImage=chrome
OpenUPnP=y
Template=RpcPortBindings
Template=SpecialImages
Template=COM
Template=OpenCOM
Template=BoxedCOM
Template=RpcPortBindingsExt
```
I don't know whether all of these are necessary. There are almost certainly some that aren't. But with this config, it works, so I'm not questioning it.

**Reply by e-t-l** [reply]
Btw, it sounds like you do a fair amount of gaming within Sandboxie, so how is your graphics performance like? There's a number of other open issues right now about GPU passthrough not working. No idea how having an older OS, and presumably older hardware, might affect that. Have you run any graphics benchmark inside Sandboxie to compare it to your unsandboxed graphics performance?

**Reply by ghost** [reply]
The games I have listed above don't feature any benchmark tool. The only games that do are Unreal Gold and Unreal Tournament GOTY. I did run a benchmark in both of them but difference in performance was so insignificant that it wasn't even worth reporting it. I can remember that a short benchmark demo of Unreal Gold launched in yellow box provided 490fps while in green box it was 500fps. However, this is an area I am going to explore in future as it is definitely important for gaming use-case in Sandboxie.

**Reply by e-t-l** [reply]
> don't feature any benchmark tool

Right, sorry I should have been more specific. I meant just a normal GPU benchmarking program that could be downloaded and run in the sandbox

> in yellow box provided 490fps while in green box it was 500fps

What about not sandboxed at all? I ask because there are at least some programs currently that can't properly utilize graphics, even in a green sandbox. I was just curious whether your Steam games suffered at all from a similar problem.

**Reply by ghost** [reply]
Per your request I did a benchmark with _Operation Flashpoint: Cold War Crisis_ (appears in the list in the original post). I used a benchmark-alike scenario made by a community member so it is not 100% accurate but it does render performance tendency. The results are devastating. The game outside Sandboxie gets 5000 points total while sandboxed (yellow or green) only 2500. It means the sandboxed game gets 50% performance hit.
However, I don't think the results are representative with other games as sandbox performance seems to be per-game specific. It will require further research to come up with any conclusion.

**Reply by ghost** [reply]
I've done another test. This time with _GRID: Autosport_ which has built-in benchmark feature. So far I've tested it in Sandboxie only, as I haven't installed the game in my OS yet. The results are:<hr>YELLOW BOX
<fps_total min_fps="83.408981" max_fps="113.926949" av_fps="101.507591" min_fps_ms="11.989117" max_fps_ms="8.777555" av_fps_ms="9.851480" />
GREEN BOX
<fps_total min_fps="115.546089" max_fps="203.296005" av_fps="150.188889" min_fps_ms="8.654555" max_fps_ms="4.918936" av_fps_ms="6.658282" /><hr>As you can see, the tendency is unfortunately similar and relatable to the benchmark made in the previous game.
- min_fps: GREEN BOX 39% more
- max_fps: GREEN BOX 78% more
- av_fps: GREEN BOX 49% more

**Reply by ghost** [reply]
Just to make sure I've made another benchmark. This time in GRID2 (released in 2013) which was made one year prior to GRID: Autosport but both share the same engine, technology and requirements. Again, tested only in Sandboxie.<hr>YELLOW BOX
<fps_total min_fps="105.671944" max_fps="159.882614" av_fps="129.760391" min_fps_ms="9.463250" max_fps_ms="6.254589" av_fps_ms="7.706512" />
GREEN BOX
<fps_total min_fps="137.505951" max_fps="237.419540" av_fps="173.295853" min_fps_ms="7.272412" max_fps_ms="4.211953" av_fps_ms="5.770479" /><hr>The differences in the results are a little bit smaller comparing to the benchmark in Autosport but in general comply with the performance tendency.
- min_fps: GREEN BOX 30% more
- max_fps: GREEN BOX 49% more
- av_fps: GREEN BOX 34% more

**Reply by isaak654** [reply]
See #3977 for increased FPS with Cyberpunk 2077 (standard sandbox).

**Comment by ghost** [comment]
Steam definitely works within Sandboxie, but not connect games to Valve's Anti-Cheating enabled servers due "invalid signature".

**Comment by ghost** [comment]
Has anyone been banned by BattlEye for running Ubisoft in Sandboxie (without any cheating of any kind)?

**Comment by DavidXanatos** [comment]
@Mitch037 can we make this a permanent page in the docs repository?

**Reply by ghost** [reply]
Yes

**Reply by DavidXanatos** [reply]
great!
would you like to do it and submit a pull request ?

**Reply by ghost** [reply]
> great! would you like to do it and submit a pull request ?

I have no idea how to do it. Could you tell me what I have to do in order to submit it?

**Reply by DavidXanatos** [reply]
fork the docs repo add a *.md file and put the table in int, once committed to your fork github will show you an optionto create a pull request as a bar above the sources

**Reply by ghost** [reply]
> fork the docs repo add a *.md file and put the table in int, once committed to your fork github will show you an optionto create a pull request as a bar above the sources

done

**Comment by DavidXanatos** [comment]
continue at: https://github.com/sandboxie-plus/Sandboxie/discussions/4172
